src/main/java/org/cryptomator/cryptofs/health/dirid/EmptyDirFile.java

```diff
@@ -0,0 +1,45 @@
+package org.cryptomator.cryptofs.health.dirid;
+
+import org.cryptomator.cryptofs.VaultConfig;
+import org.cryptomator.cryptofs.health.api.DiagnosticResult;
+import org.cryptomator.cryptolib.api.Cryptor;
+import org.cryptomator.cryptolib.api.Masterkey;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Map;
+
+import static org.cryptomator.cryptofs.health.api.CommonDetailKeys.DIR_ID_FILE;
+
+public class EmptyDirFile implements DiagnosticResult {
```

Comment:
Add some javadoc explaining that empty dir ids are reserved for the root dir only.